public/src/views/FormView.jsx

```diff
@@ -31,7 +33,7 @@ export const FormView = () => {
       accident,
       sleep,
       awoke,
-      datetime,
+      datetime: formatDate(datetime).replace('T', ' '),
```

Comment:
I've seen this twice now. You should have this logic in the formatDate or provide another helper (formatDateTime?)

Reply:
Fixed!